### PR TITLE
docs(README): remove coffeescript from README

### PR DESCRIPTION
**Pull request by ariporad**
Other than the CoffeeScript section

@nfischer: Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ if (exec('git commit -am "Auto-commit"').code !== 0) {
 
 ### CoffeeScript
 
+CoffeeScript is also supported automatically:s
```
Comment (nfischer) [inline README.md]: I think ":s" on the end is a typo

File: README.md [hunk 2]
```diff
@@ -106,31 +108,39 @@ shell.echo('hello world');
...
+  cd(__dirname);
+  mkdir('docs');
+  cd('lib');
+  var files = ls('*.js');
+  for(var i = 0; i < files.length; i++) {
+    var text = grep('//@', file);     // extract special comments
+    text = text.replace('//@', '');   // remove comment tags
+    text.to('docs/my_docs.md');
+  }
+};
```
Comment (nfischer) [inline README.md]: I haven't tested this example, but if you think it works, it should be good enough for me.

Comment (ariporad) [inline README.md]: Whoops, that looks wrong. Let me fix that.

File: README.md (hunk 1 above)
Comment (ariporad) [inline README.md]: Yeah. I think it is too.

File: README.md (hunk 2 above)
Comment (ariporad) [inline README.md]: @nfischer: I can't figure out why its not working, but `replace` ***ABSOLUTELY*** refuses to replace more than one instance of something. We'll have to hold off on this till 0.7.0.

Comment (nfischer) [inline README.md]: I think you'll need to use a regex as the argument to replace, and supply the `g` flag.

```
text.replace(/\/\/@/g, '');
```

Comment (ariporad) [inline README.md]: !!!!!

**Comment by ariporad** [comment]
@nfischer: Ok, everything should be fixed now.

**Comment by nfischer** [comment]
LGTM

**Comment by ariporad** [comment]
I'll merge once Travis finishes.

**Pull request merged**